### ISSUE TITLE
Simple, example IU-HRV script not downloading correctly

### ISSUE THREAD

**Issue by lsawade** [PyGLImER/PyGLImER#98]
Hi,

I have been trying to figure out the sphinx gallery thing, and it's actually pretty cool. Examples are being run correctly matplotlib figure are automatically rendered etc.

BUT I can't figure out what's going on with the data download in download_small_db. It almost seems like it's in an infinite loop. And then, returns no waveforms at all.

Here a preliminary script:
https://github.com/PyGLImER/PyGLImER/blob/devel-lucas/examples/tutorials/DataCollectionMSEED.py

**Comment by PeterMakus** [comment]
Hey I will look into this today :)

**Comment by lsawade** [comment]
> Ok, this is fixed now in the `dev` branch.
> 
> For your test script, note that it's not necessary to create folders - PyGLImER takes care of that.

I know it usually does! But it the code failed because it couldn't write stationxmls because the stations folder didn't exist. The massdownloader creates this folder automatically, but the normal bulk request does not!

**Comment by PeterMakus** [comment]
Yeah, sorry I forgot about the station folder. I just added a line for that today
